packages/core/src/app/shipping/shippingOption/ShippingOptionsForm.tsx

```diff
@@ -33,178 +31,166 @@ const getShippingOptionIds = ({ consignments }: ShippingOptionsFormProps) => {
     return { shippingOptionIds };
 };
 
-class ShippingOptionsForm extends PureComponent<
-    ShippingOptionsFormProps & FormikProps<ShippingOptionsFormValues>
-> {
-    private unsubscribe?: () => void;
+function getRadioInputName(consignmentId: string): string {
+    return `shippingOptionIds.${consignmentId}`;
+}
+
+export interface ShippingOptionsFormValues {
+    shippingOptionIds: {
+        [shippingOptionIds: string]: string;
+    };
+}
 
-    componentDidMount(): void {
-        const { subscribeToConsignments } = this.props;
+const NoShippingOptions: React.FC<{ message: React.ReactNode }> = ({ message }) => (
+    <div className="shippingOptions-panel optimizedCheckout-overlay">
+        <p
+            aria-live="polite"
+            className="shippingOptions-panel-message optimizedCheckout-primaryContent"
+            role="alert"
+        >
+            {message}
+        </p>
+    </div>
+);
 
-        this.unsubscribe = subscribeToConsignments(this.selectDefaultShippingOptions);
-    }
+const ConsignmentDetails: React.FC<{ cart: any; consignment: Consignment }> = ({ cart, consignment }) => (
```

Comment:
Can we move these to a separate file. Also since there are no children, we can use ReactNode.

Also can we avoid any for `cart` ?